### PR TITLE
Adding CDNC

### PR DESCRIPTION
**Pull request by jthomson1**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -2115,5 +2115,20 @@
...
+    "social":{
+      "facebook":"",
+      "twitter":"https://twitter.com/CDNCardano",
+      "website":"https://www.cdncardano.com/index.html",
+      "reddit":"",
+      "instagram":"",
+      "youtube":""
+    },
+    "telegram":"cdncardano",
+    "operatorName":"CDNC Canadian Cardano",
```
Comment (leo42) [inline registry.json:2131]: Please use a real name or leave this empty.

**Review by leo42 — CHANGES_REQUESTED** [review]

**Comment by jthomson1** [comment]
For operator name? We are a Canadian company. Sorry for the confusion. We
are a group of four individuals operating as "CDNC Canadian Cardano". Do
you need a human name?

Thanks!

**Comment by leo42** [comment]
The operator name is for Operators that want to be eponymous (instead of anonymous) .

It is meant to reference a real person not a legal entity.  you an leave it empty if you like(it is optional)

**Comment by jthomson1** [comment]
Ok, thanks for the clarification. I'll resend.

**Pull request closed without merging**